components/posts/data.js

```diff
@@ -0,0 +1,33 @@
+const bearer = "joDiolo3-TL5MGpDG2TVAOSn-EN5Z3G3qXYpKuCg-l0";
+
+export async function getPhPosts() {
+  var params = {
+    headers: {
+      Authorization: "Bearer " + bearer
+    }
+  };
+
+  return fetch("https://api.producthunt.com/v1/posts", params)
+    .then(res => res.json())
+    .then(
+      result => {
+        return parsePhPosts(result.posts);
+      },
+      error => {
+        return {
+          error
+        };
+      }
+    );
```

Comment:
How does this look? https://github.com/mbinet/topfive/pull/1/commits/042bee24853454b75cbc82ad4c23d9b886bf2fbb
I used async/await + created a wrapper to parse the res in json as suggested.

Also I realised the error management was handled implicitly so that's cool